src/features/high-fee-drawer/high-fee-drawer.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+import { FiAlertTriangle } from 'react-icons/fi';
+import { Box, color, Stack } from '@stacks/ui';
+
+import { useDrawers } from '@common/hooks/use-drawers';
+import { ControlledDrawer } from '@components/drawer/controlled';
+
+import { HighFeeConfirmation } from './components/high-fee-confirmation';
+
+export function HighFeeDrawer(): JSX.Element {
+  const { showHighFeeConfirmation, setShowHighFeeConfirmation } = useDrawers();
+
+  useEffect(() => {
+    return () => {
+      if (showHighFeeConfirmation) setShowHighFeeConfirmation(false);
+    };
+  }, [showHighFeeConfirmation, setShowHighFeeConfirmation]);
+
+  return (
+    <ControlledDrawer
+      title={<Box as={FiAlertTriangle} color={color('feedback-error')} size="36px" />}
+      isShowing={!!showHighFeeConfirmation}
+      onClose={() => setShowHighFeeConfirmation(false)}
+    >
+      <Stack px="loose" spacing="loose" pb="extra-loose">
```

Comment:
I don't think `<Stack />` is the right element to use here. It's for spacing multiple elements, yet here we have one. Could we use a Box?

Reply:
This is needed inside `HighFeeConfirmation` so I'll move it there.